lib/secure_headers.rb

```diff
@@ -190,17 +270,19 @@ def header_classes_for(request)
         ALL_HEADER_CLASSES
       else
         HTTP_HEADER_CLASSES
-      end
+      end.map do |klass|
```

Comment:
I was about to comment on this and say a similar thing about the readability of this 😛  The `def blah; end.map do |klass|` definitely makes for it slightly difficult to follow. Is there an approach that we could take here to break this up a bit?